### PR TITLE
adding chapter level sponsors, fixes #171

### LINKED ISSUE #171: Add chapter level sponsors to chapter pages.
Currently, chapter sponsors do not show up on the chapter level pages. This will take a bit of design help for layout + template additions to be able to make this work. Very design friendly!

### PR DESCRIPTION
**Pull request by minamarkham**

### CODE REVIEW

**Comment by kstack7** [comment]
reviewing :+1:

**Comment by kstack7** [comment]
@Cara-Jo do you want to play with the sponsor links a bit? @minamarkham what do you think about nixing the web addresses all together?

Philly's sponsors made me think we could tweak this a bit to keep the design/layout consistent: 
screen shot 2015-02-15 at 11 31 24 pm

**Comment by minamarkham** [comment]
I'm good with removing the websites/names altogether.

File: app/views/chapters/show.html.erb [hunk 1]
```diff
@@ -13,20 +13,25 @@
...
 
-      <!-- <% if @chapter.sponsors.count != 0 %>
+      <% if @chapter.sponsors.count != 0 %>
         <div class="hide">
           <div class="sponsors">
             <h2>Sponsors</h2>
-            <div id="logos">
+            <div class="logos">
               <% @sponsors.each do |sponsor| %>
                 <% if sponsor.image? %>
```
Comment (juliaelman) [inline app/views/chapters/show.html.erb]: We should probably add an `<% else %>` statement in here for instances where there is no sponsor logo.

**Comment by kstack7** [comment]
^ Yes, @juliaelman !

**Comment by minamarkham** [comment]
Okay, I'll change it to an if/else statement for logo or sponsor name. Right now, I've coded it to show the sponsor name underneath the logo, which I think is defaulted to the URL.

**Comment by juliaelman** [comment]
@minamarkham sweet biz! Let us know when those changes are pushed up and we can review :dart:

**Comment by minamarkham** [comment]
@kstack7 thoughts on this one?

File: app/views/chapters/show.html.erb (hunk 1 above)
Comment (kstack7) [inline app/views/chapters/show.html.erb]: hey @minamarkham can we get a target="_blank" in the sponsor url? Otherwise things look good!

**Comment by minamarkham** [comment]
Pushed!

**Comment by kstack7** [comment]
Looks good @minamarkham ! :rocket:

**Pull request merged**